design-system-docs/src/_component_examples/_checkbox/default.erb

```diff
@@ -3,5 +3,5 @@ title: default
 ---
 
 <%=  render CitizensAdviceComponents::CheckboxSingle.new(name: "a-single-checkbox") do |c|
-  c.checkbox({ label: "I agree to the great terms and conditions", value: "agreed-to-tcs" })
+  c.checkbox(label: "I agree to the great terms and conditions", value: "agreed-to-tcs")
```

Comment:
See https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/

Ruby 3 separates how positional and keyword arguments are handled. Passing in arguments to the single checkbox as a hash relied on a quirk of ruby 2 where hashes were converted to positional arguments.

Now we either need to pass them explicitly or use the ** double splat operator. It's simplest to use the named parameters directly here.